lib/classes/Variables.js

```diff
@@ -17,6 +17,7 @@ class Variables {
     this.envRefSyntax = RegExp(/^env:/g);
     this.optRefSyntax = RegExp(/^opt:/g);
     this.selfRefSyntax = RegExp(/^self:/g);
+    this.importsRefSyntax = RegExp(/^imports:/g);
```

Comment:
This source is not needed imo. Adds lots of technical debts. Lets just use "self"